python/bin/xpy_dev_open

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+
+import xnvme.ctypes_mapping as capi
+
+capi.guard_unloadable()
```

Comment:
I think it's better to move this to the library.

Reply:
How about this would go on the list of things to refine in ``docs/python/index.rst``? Have done so now.